src/strings/en-us.json

```diff
@@ -826,8 +826,8 @@
     "LabelSaveLocalMetadataHelp": "Saving artwork into media folders will put them in a place where they can be easily edited.",
     "LabelScheduledTaskLastRan": "Last ran {0}, taking {1}.",
     "LabelScreensaver": "Screensaver:",
-    "EnableFastImageFadeIn": "Fast Image Fade Animations",
-    "EnableFastImageFadeInHelp": "Show posters and other images with a quicker fade animation when they finish loading.",
+    "EnableFastImageFadeIn": "Faster animations",
```

Comment:
Also, a little bit outside the scope of this PR, but I decided to change those because we're no longer talking about images, but also the placeholder below, as this toggle controls the speed of blurhashes and images animations (whatever you have enabled). We've been also talking many times about the animation times of other UI components and the possibility of making them configurable, etc... in Matrix. Until we get to the point where we make that configurable, I think a single (and straight to the point) checkbox to rule them all is the best idea.

Reply:
It'd be good to rename the key for translators as well :)